packages/xo-web/src/xo-app/sr/tab-disks.js

```diff
@@ -304,6 +313,7 @@ class NewDisk extends Component {
 @connectStore(() => ({
   checkPermissions: getCheckPermissions,
   vbds: createGetObjectsOfType('VBD'),
+  vmsSnapshotsBySuspendedVdi: createGetObjectsOfType('VM-snapshot').groupBy('suspendVdi'),
```

Comment:
There should only be one snapshot per suspend VDI, right? So you can use `keyBy` instead.